packages/synapse-react-client/src/components/AccessTokenPage/AccessTokenPage.tsx

```diff
@@ -21,64 +21,55 @@ export const AccessTokenPage: React.FunctionComponent<AccessTokenPageProps> = ({
     data: infiniteData,
     isLoading,
     error,
-    refetch,
     fetchNextPage,
     hasNextPage,
   } = useGetPersonalAccessTokensInfinite()
 
-  // We rerender the list whenever a token is created or deleted to ensure we are up-to-date
-  const rerenderList = () => {
-    refetch()
-  }
```

Comment:
No longer necessary since our custom hooks invalidate the react-query cache and automatically trigger a re-fetch when appropriate